staging/src/k8s.io/pod-security-admission/test/fixtures_seccomp_baseline.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/pod-security-admission/api"
+)
+
+/*
+Note: these fixtures utilize seccomp helper functions that ensure consistency across the
+alpha annotation (up to v.1.19) and the securityContext.seccompProfile field (v1.19+).
+
+The check implementation looks at the appropriate value based on version.
+*/
+
+func init() {
+	fixtureData_baseline_1_0 := fixtureGenerator{
+		expectErrorSubstring: "seccomp profile",
+		generatePass: func(p *corev1.Pod) []*corev1.Pod {
+			if val, ok := p.Annotations[annotationKeyPod]; ok && val == corev1.SeccompProfileRuntimeDefault {
+				return []*corev1.Pod{p}
```

Comment:
https://github.com/kubernetes/kubernetes/pull/103517 merged, can return nil here now (also add a comment about what this is doing... "// the minimal valid pod already had to set seccomp options")